client/src/shared/navigation/desktop.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Box } from '@material-ui/core';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import BookmarkBorderIcon from '@material-ui/icons/BookmarkBorder';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Link as RouterLink } from 'react-router-dom';
+import t from 'locale/he_IL';
+import logo from 'assets/logo.png';
+import { Button, Row, IconButton, Menu } from 'shared';
+import { colors } from 'style/index'
+import * as SC from './style'
+import { useDispatch } from 'react-redux';
+import { openModal } from 'redux/modal/slice';
+
+const DesktopNavBar = ({ user, isAuthenticated, logoutHandler }) => {
+	const dispatch = useDispatch()
+	const [dropDownEl, setDropDownEl] = React.useState(null);
+	const handleDropDownClick = (event) => {
+		setDropDownEl(event.currentTarget);
+	};
+	const handleDropDownClose = () => {
+		setDropDownEl(null);
+	};
+	const dropdownItems = [{ 'text': 'התנתק', 'onClick': logoutHandler }]
```

Comment:
Why not in the const file? or inthe t file

Reply:
There's a property exist in the HE_IL file called "signout", in Hebrew we might use the word התנתקות instead of התנתק